src/components/gui/gui.jsx

```diff
@@ -18,6 +19,13 @@ import MenuBar from '../menu-bar/menu-bar.jsx';
 
 import styles from './gui.css';
 
+const layout = {
+    fullStageWidth: 480,
+    fullStageHeight: 360,
+    smallerStageWidth: 480 * 0.85,
+    smallerStageHeight: 360 * 0.85,
+    fullSizeMinWidth: 1096
+};
```

Comment:
From offline: Can we put these somewhere that can be imported from elsewhere? Or maybe make a wrapped `MediaQuery` component that has them embedded?